library/network/src/lib/network/susefirewalld.rb

```diff
@@ -815,7 +815,7 @@ def GetIgnoreLoggingBroadcast(_zone)
     #	// Do not log broadcast packetes from DMZ
     #	SetIgnoreLoggingBroadcast ("DMZ", "yes")
     def SetIgnoreLoggingBroadcast(_zone = nil, bcast)
-      bcast = bcast.downcase == "no" ? "broadcast" : "off"
+      bcast = bcast.casecmp("no").zero? ? "broadcast" : "off"
```

Comment:
to be honest I am not sure if `bcast.casecmp("no").zero?` have at least same readability as `bcast.downcase == "no"`